sample/testRecorder-complete/pages/product.js

```diff
@@ -0,0 +1,36 @@
+module.exports = createPageObjects({
+  Product: {
+    actions: {
+            
+    },
+    assertions: {
+      theProductTitleIsShown: function () {
```

Comment:
this is the object header, which contains a title, so more like theProductIsShown